### DISCUSSION TITLE
suddenly a "cp -a (to /run/initramfs/ltsp/[imagename])" appears within every boot and then ends in "login incorrect"

### DISCUSSION THREAD

**Discussion by jorigel** [ltsp/ltsp#314] [Q&A]
Hi,
maybe these are two errors, but happen all over sudden from one day to the other out of a working system. The system copies the image any time it boots to the upper path (takes rather long). When continuing to the boot screen it shows correct users but don't accept a login with "login incorrect".

---begin
root@nuc~ > ltsp info
LTSP version: 20.06-1~ubuntu18.04.1
SERVER XSESSIONS: mate.desktop
SERVER OS: PRETTY_NAME="Debian GNU/Linux 10 (buster)"
CHROOTS:

VMs:
xubuntu

IMAGES:
ubuntu-mate-20.04.1
xubuntu
xubuntu-20.04.1
--- end

I already tried to refresh the user information by "ltsp initrd" but that didn't help.

**Comment by alkisg** [comment]
> The system copies the image any time it boots to the upper path

It's not supposed to do that. Try removing your ltsp.conf completely, in case you have something there that breaks things; remember to run `ltsp initrd`.

Also if you can, upload a screenshot of the `cp -a ...initramfs` error that you mentioned.

**Comment by abixadamj** [comment]
Please provide `/etc/ltsp/ltsp.conf` file here .. maybe you have `IMAGE_TO_RAM=1` ?

**Comment by abixadamj** [comment]
@jorigel - How is your problem?

**Comment by jorigel** [comment]
Hi abixadamj,
I followed Alkis advice and solved the problem that way, but I still haven't found the reason why that happened all over sudden. 
ThanX for Your help as well.
jori
